### PR TITLE
add xcm transactor v3 docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds support and information on the XCM Transactor V3 precompile. 

I tried a new thing to split up the V2 and V3 docs, where I put them into tabs and had text inside of the tabs. We don't typically put plain text inside of tabs, but with the new style of tabs we can easily make a minor css change to make it look good. I'll wait til I get feedback to open a PR in mkdocs, but I did a quick edit in the browser so you can say aye or nay on it 🙂 

Screenshot 2023-09-06 at 10 32 04 PM

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

Chinese PR for variable name changes: https://github.com/moonbeam-foundation/moonbeam-docs-cn/pull/336

### After Translation Requirements

- [x] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/interoperability/xcm/xcm-transactor.md [hunk 1]
```diff
@@ -193,63 +193,94 @@ Once the transaction is processed, Alice should've received one token in her add
...
+    - **transactThroughSigned**(*Multilocation* *memory* dest, *address* feeLocationAddress, *uint64* transactRequiredWeightAtMost, *bytes* *memory* call, *uint256* feeAmount, *uint64* overallWeight) — function that represents the `transactThroughSigned` method described in the [previous example](#xcmtransactor-transact-through-signed), setting the **fee** type to **AsCurrencyId**.  Instead of the asset ID, you'll need to provide the [asset XC-20 address](/builders/interoperability/xcm/xc20/overview/#current-xc20-assets){target=_blank} of the token that is used for fee payment
+    - **encodeUtilityAsDerivative**(*uint8* transactor, *uint16* index, *bytes memory* innerCall) - encodes an `asDerivative` wrapped call given the transactor to be used, the index of the derivative account, and the inner call to be executed from the derivated address
+
+=== "V3 (Moonbase Alpha only)"
```
Comment (albertov19) [inline builders/interoperability/xcm/xcm-transactor.md:266]: I don't mind this but can we add a bit of margin here plz

image

**Review by albertov19 — CHANGES_REQUESTED** [review]
Looks good! I pushed a commit with some edits, please check.

I also left a comment

**Comment by eshaben** [comment]
@albertov19 I merged the PR that adds a margin. This PR is ready to go, just needs one last review, please!

**Review by albertov19 — APPROVED** [review]
LGTM!

**Pull request merged**